source/manual/email-notifications-how-they-work.html.md

```diff
@@ -9,68 +9,53 @@ parent: "/manual.html"
-* Provides an HTTP JSON API to alter stored data, such as: create alerts,
-  set up lists and set up subscription preferences.
-* Coordinates sending of notifications with the underlying email sending
-  service, which is currently GOV.UK Notify.
+- [Technical dashboard][email-alert-api-technical]
+- [Product dashboard][email-alert-api-product]
+- [How we trigger emails][email-alert-api-matching]
 
-Frontend applications use Email Alert API to create new mailing lists and
-as a redirect destination for signing up. Publishing apps tend not to use
-email alert API, instead relying on Email Alert Service to be triggered
-by Publishing API. The exceptions are [Travel Advice Publisher] and
-[Specialist Publisher], which require a greater degree of control, so
-communicate directly with the Email Alert API.
+## Email system apps
 
-To have a near real-time overview of the status of data passing through
-the Email Alert API, view the [metrics dashboard][dashboard].
+- [Finder Frontend][finder-frontend-email]
+  - Provides a UI to sign up to filterable search results.
 
-For more information about how content changes trigger emails, as well as
-common issues and troubleshooting, view [Email Alert API docs].
+- [Travel Advice Publisher] / Specialist Publisher
+  - Use Email Alert API directly to send email ([tech debt][travel-specialist-tech-debt]).
 
-### Email Alert Frontend
+- [Email Alert Frontend][email-frontend-readme]
+  - Provides a UI to sign up to static pages e.g. a taxon.
+  - Provides a UI for a user to manage their subscriptions.
+  - Communicates with Email Alert API to make changes.
 
-* Provides a user interface to sign up to a mailing list.
-* Provides a user interface for a user to manage their subscriptions.
-* Communicates with Email Alert API to make changes.
+- [Email Alert Service][email-service-readme]
+  - Listens to the Publishing API message queue for `major` changes to content.
+  - Communicates with Email Alert API API to trigger sending emails.
 
-### Email Alert Service
+- [Email Alert API][email-api-readme]
+  - Stores data about subscribers and the emails we send to them.
+  - Provides APIs for working with subscriptions and sending emails.
+  - Sends email using GOV.UK Notify, and [deals][email-spam-report] with spam reports.
 
-* Listens to the Publishing API message queue.
-* Communicates with Email Alert API when an appropriate Publishing API
-  event occurs to trigger an alert. This is usually a ‘major’ edit to
-  some content.
+Communication from Email Alert API to Notify is done via a HTTP API which is authenticated by [an API key][notify-api-key]. Communication from Notify to Email Alert API is [authenticated][email-spam-auth] with Signon and a bearer token.  Email Alert API is an internal application, so to enable callbacks two endpoints are [exposed][email-spam-public] publicly through <https://email-alert-api-public.publishing.service.gov.uk> (similarly for Integration and Staging).
```

Comment:
This isn't quite true unfortunately, accounts have temporarily turned the whole app to public again. But probably not worth documenting that oddity (and hopefully it goes away soon).